devel/spicy/getting-started.rst

```diff
@@ -56,16 +56,14 @@ We do both of these by creating an additional control file for Zeek:
     :linenos:
     :language: spicy-evt
 
-The first block (lines 1-3) tells Zeek that we have a new protocol
-analyzer to provide. The analyzer's Zeek-side name is
-``spicy::MyHTTP``, and it's meant to run on top of TCP connections
-(line 1). Lines 2-3 then provide Zeek with more specifics: The entry
+The first block (lines 1-2) tells Zeek that we have a new protocol
+analyzer to provide. Line 1 defines the analyzer's Zeek-side name as
+``spicy::MyHTTP`` and that it's meant to run on top of TCP connections.
+Line 2 provide Zeek with more specifics: The entry
```

Comment:
```suggestion
Line 2 provides Zeek with more specifics: The entry
```

I would actually just rewrite this line entirely. The second part was removed so it's no longer a list of specifics being provided.

> Line 2 provides Zeek with the entry point...